src/App.js

```diff
@@ -0,0 +1,18 @@
+import "./App.css";
+import Header from "./Header/Header";
+import Main from "./Main/Main";
+import Section from "./Section/Section";
+import Footer from "./Footer/Footer";
+
+function App() {
+  return (
+    <div>
+      <Header />
+      <Main />
+      <Section />
+      <Footer />
+    </div>
```

Comment:
[중요도: 중간]
불필요한 div는 제거해도 괜찮습니다.
```suggestion
    <>
      <Header />
      <Main />
      <Section />
      <Footer />
    </>
```
<></>는 Fragment로 React에서 하나의 컴포넌트 return할 때 하나의 큰 element로 return해야하는데 준용님 코드처럼 한번에 하나의 큰 element를 return하지 못할 때 이 Fragment로 묶어 return하게 해줄 수 있습니다.

Fragment는 실제 html에 보여줄 때 해당 태그를 제거해줍니다.